.gitignore

```diff
@@ -26,27 +26,28 @@ Makefile*
 *-build-*
 
 # QtCreator
-
+CMakeLists.txt.user
 *.autosave
 
-#QtCtreator Qml
+# QtCtreator Qml
 *.qmlproject.user
 *.qmlproject.user.*
 
-#Linux libs
-*.so.*
+# VSCode
+.vs
 
-#Libs
-!libs/**
+# Linux libs
+*.so.*
 
-#Binary Output
+# Build folder and Binary Output
+/build
```

Comment:
Add /out folder to gitignore
Visual Studio 2019 and 2022 builtin cmake support places config and build artefacts into /out/%config_name%/*